pkg/repositories/models/workflow.go

```diff
@@ -13,7 +13,7 @@ type Workflow struct {
 	BaseModel
 	WorkflowKey
 	TypedInterface          []byte
-	RemoteClosureIdentifier string `gorm:"not null"`
+	RemoteClosureIdentifier string `gorm:"not null" valid:"length(3|50)"`
```

Comment:
i dont know what this string it - @katrogan

Reply:
this is the path to where on s3 (or whichever cloud blobstore) we upload the workflow closure